### ISSUE TITLE
Video filling the entire screen?

### ISSUE THREAD

**Issue by dragons4life** [YePpHa/YouTubeCenter#1438]
<img src'http://a.pomf.se/qgnznd.png'/>

Does this happens to anyone else, is there a way to fix it? it only happens to videos that do not have a 6:9 aspect ratio if i had to guess?

**Comment by YePpHa** [comment]
There's an option called `Player aspect` in the settings under Player -> Player Size, which you can set to 16:9 and YouTube Center will force the player into a 16:9 aspect ratio.

**Comment by dragons4life** [comment]
Thank you a lot.

**Comment by YePpHa** [comment]
@dragons4life yes, it's different. The aspect button changes the video's aspect and not the players.
